packages/puppeteer-extra-plugin-stealth/evasions/user-agent-override/index.js

```diff
@@ -35,7 +35,6 @@ const { PuppeteerExtraPlugin } = require('puppeteer-extra-plugin')
  * @param {Object} [opts] - Options
```

Comment:
The example code in the comment higher up (can't select that) still shows `platform` which we should remove and maybe add a note that it'll be derived/spoofed based on the UA string (while we're at it: we should remove `q=0.9` from the `locale` example as that isn't supported)

Reply:
PS: After updating jsdoc comments we should run `yarn docs-for-evasions && yarn postdocs-for-evasions` in the stealth dir

Reply:
Thanks, I missed that! Fixed in 12a76f2